src/objectsAdvanced/task1.js

```diff
@@ -0,0 +1,5 @@
+const fn = (obj) => {
+    for (let key in obj)
```

Comment:
Loops and if expressions - always must have curly brackets { }
And consider using semicolon at the end of expression ;
And try to use literal expression in console.log instead of +